resources/builder.inc.sh

```diff
@@ -1369,17 +1378,26 @@ _builder_parse_expanded_parameters() {
           # internal use parameter for dependency builds - identifier of parent script
           shift
           builder_dep_parent="$1"
+          builder_echo setmark "dependency build, started by $builder_dep_parent"
+          builder_echo grey "build.sh parameters: <${_params[@]}>"
```

Comment:
Moves the reporting ahead of any other messages that might appear, see also ll.1386, for example debug messages on ll.1397 should appear after the note saying that we are a child build or dep build.